### PR TITLE
Incubator releases

### PR DESCRIPTION
**Pull request by justinmclean**
Talk on making release while in the incubator.

### CODE REVIEW

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+* You may want to make that clear in the LICENSE file
+
+== License File
+* License file named LICENSE or LICENSE.txt in the root directory
+* Contains Apache license and list of licenses of bundled software in a distribution
+* Short form pointer to license preferred
+* May have different contents for source and binary
+
+== License File
+image::LICENSE.png[]
```
Comment (soenkeliebau) [inline content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: File missing?

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 2]
```diff
@@ -0,0 +1,409 @@
...
+* Contains ASF copyright and developed at the ASF notice
+* Keep year up to date
+* Only add what is needed
+** Relocated copyright notices
+** Required notices from bundled Apache software notice files
+** All other required notices
+* In general licensing info shouldn’t be in the notice file
+
+== Notice file
+image::NOTICE.png[]
```
Comment (soenkeliebau) [inline content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: File missing?

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 3]
```diff
@@ -0,0 +1,409 @@
...
+
+== Required Notices
+* Some confusion to what is a required notice
```
Comment (soenkeliebau) [inline content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: Sounds off to me, but not a native speaker.

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+
+== Binary Distributions
+* Not considered an official release
+* Need to comply the same way with policies as source distributions
+* LICENSE and NOTICE may be different as the content of the release is often different from the source release
+
+== Where to Ask for Help
+* Ask on your mailing lists
+* Ask your mentors
+* Email the incubator mailing list at:+
```
Comment (soenkeliebau) [inline content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: errant +

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 5]
```diff
@@ -0,0 +1,409 @@
...
+=== Only Dead People
+video::deadpeople.mp4[]
+
+=== GPL or BSD?
+video::both.mp4[]
+
+== Documentation Issues vs Errors
+* It’s better to have a documentation issue than a licensing error
+* Minor issues are OK to be fixed in the next release
+* If you unsure a license should be added add it
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: fixed

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 6]
```diff
@@ -0,0 +1,409 @@
...
+* I’m a volunteer and not paid to do this. Not even by my day job
+* This talk contains my views and may not represent the incubator as a whole
+endif::[]
+
+== What is the Apache Incubator?
+* Where communities come to learn the Apache Way
+* Likes existing projects with a community around them
+* Main entry point for new projects
+
+== Why we Have an incubating Process?
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: fixed

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 7]
```diff
@@ -0,0 +1,409 @@
...
+* Podling follows the Apache Way
+* Ensure donations comply with the Apache 2.0 license
+* Podling follows the ASF structure of contributors, committers and PMC members
+* Podling grants more responsibility via meritocracy
+* Ensure that decision making is done in the open
+* People act as individuals, not the company they work for
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: I don't think, it's related, but it's one of the things podlings struggle with during incubator

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 8]
```diff
@@ -0,0 +1,409 @@
...
+
+== The Apache Way
+* Charity - For the public good. Software costs nothing
+* Pragmatic - Business friendly
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: Why not? The license allows for commercial use without giving back. See http://theapacheway.com/pragmatic/ for more detail.

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc [hunk 9]
```diff
@@ -0,0 +1,409 @@
...
+* Community - Collaboration, consensus and diversity
+* Merit - The more you do you more responsibility you have. Not just code contributions
+* Open - Everything in the public view. Discussions occur on mailing lists. Everyone can participate
+* Consensus - Work together to find ways forward
+
+== Apache License
+* Permissive license - you can do what you want
+* Business friendly - can be used for commercial projects
+* Requires source headers, a LICENSE and (optionally) a NOTICE file
+* If your not an ASF project:
```
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: fixed

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc (hunk 3 above)
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: rewrote

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc (hunk 4 above)
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: Pluses are needed for line continuation. I made it consistent with others and added some where the may be required (long email address tend to break naturally).

**Comment by justinmclean** [comment]
BTW re being text heavy I agree and it's actually on purpose, it so that people can take these slides out of context of an actual presentation and still get meaningful information out of them.

File: content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc (hunk 8 above)
[the remark this answers is not in the document]
Comment (justinmclean) [reply content/ApacheWay/IncubatorReleases/src/main/asciidoc/index.adoc]: @soenkeliebau What exactly is it that you don't agree with?

**Pull request merged**